parts/linux/cloud-init/artifacts/components.json

```diff
@@ -116,7 +133,7 @@
         },
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cns",
-          "latestVersion": "v1.6.10",
+          "latestVersion": "v1.6.11",
           "previousLatestVersion": "v1.6.7",
```

Comment:
here as well?

Reply:
same. v1.6.10 version was not rolledout to all regions in aks-rp. It was rolled out only to specfic subscriptions in few regions.